src/app/space-wizard/space-wizard.component.ts

```diff
@@ -101,17 +101,16 @@ export class SpaceWizardComponent implements OnInit {
           this.host.close();
           this.reset();
         }
-        this.reset();
       },
       (err) => {
-        // TODO:consistent error handling on failures
-        let errMessage = `Failed to create the collaboration space:
-            space name :
-            ${space.name}
-            message:
-            ${err.message}
-            `;
-        alert(errMessage);
+        this.broadcaster.broadcast('toast', {
+          message: `Failed to create the collaboration space "${space.name}" because ${err.message}`,
+          type: NotificationType.DANGER
+        } as Notification);
```

Comment:
Again, just FYI... The ngx-widgets Noitification object also provides notification types. If a separate NotificationType object is desirable, we can do that.